sass/utils/_mixins.scss

```diff
@@ -126,23 +123,71 @@
 @mixin primary-button {
```

Comment:
not completely sure, but I think this mixin is used in other places in the webapp, did you verified these changes won't affect the look of the button in other places where it is used?